python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -383,7 +386,7 @@ def __init__(
             raise NotImplementedError(
                 "Read from cloud storage"
             )  # pragma: no cover; no test yet
-        if any(p.startswith("https://") for p in self.paths):
+        if any(str(p).startswith("https:/") for p in self.paths):
```

Comment:
Why the change to a single slash?

Reply:
I found it one of the scan tests failed. The path is now `PosixPath('https:/huggingface.co/datasets/scikit-learn%2Firis/resolve/main/Iris.csv')`. See the missing `/`

Reply:
Seems like this _might_ be a polars bug? Can you check?

Reply:
Yup, will do